scripts/clearoutputs.py

```
@@ -0,0 +1,27 @@
+import glob
+import sys
+from pathlib import Path
+
+current_dir = Path(os.getcwd())
+
+if input('Are you sure about deleting all files (y/n): ').strip() != 'y':
+    print('Exiting...')
+    sys.exit(0)
+
+if os.path.basename(current_dir.resolve()) != "clp-bench":
+    raise Exception("Must be run in clp-bench directory, if it was renamed, comment this line out")
```

Ruff (0.12.2)

Create your own exception

(TRY002)